tasks/tag-deploy-task.yaml

```diff
@@ -46,24 +54,29 @@ spec:
       privileged: true
     script: |
       export GIT_SSH_COMMAND='ssh -i /tekton/home/.ssh/id_rsa -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no'
-      git clone git@github.com:$(params.deploy_project_org)/$(params.deploy_project_name) /workspace/configrepo
+      git clone git@github.com:$(params.deploy_project_org)/$(params.deploy_project_name) /workspace/repo
     volumeMounts:
       - name: ssh-config
         mountPath: /tekton/home/.ssh
 
   - name: patch-imagetag
     image: $(resources.inputs.s2i-thoth.url)
-    workingDir: /workspace/configrepo
+    workingDir: /workspace/repo
     script: |
       #!/opt/app-root/bin/python
-      import yaml
 
+      import yaml
       registry_repo = "$(params.registry_project)" if "$(params.registry_project)" else "$(params.repo_name)"
-      contextpath = (
-          "$(params.overlay_contextpath)"
-          if "$(params.overlay_contextpath)"
-          else "$(params.repo_name)/overlays/test/imagestreamtag.yaml"
-      )
+
+      if "$(params.init_overlay_contextpath)" == "":
+          if "$(params.overlay_contextpath)" == "":
+              contextpath = "$(params.repo_name)/manifests/overlays/$(params.overlay_name)/imagestreamtag.yaml"
+          else:
+              contextpath = "$(params.overlay_contextpath)"
```

Comment:
Went with similar changes because:
    Case 1 (Line 77): $overlay_contextpath is empty but $overlay_name is not empty
        --> set context path based on $overlay_name since not empty
    Case 2 (Line 79) **Both** $overlay_contextpath and $overlay_name are empty
        --> use test overlay name 
    Case 3 (Line 81) $overlay_contextpath is not empty
        --> use $overlay_contextpath